assets/js/components/PageHeader/PageHeader.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+function PageHeader({ header, highlighted }) {
```

Comment:
This is super strange now :laughing: 
We should follow the children prop approach in my opinion.
Having:
```
<h1 ....>(children}</h1>
```

And using it like:
```
<PageHeader>Some text: <span className="font-bold">bold if I want</span></PageHeader>
```